pylint/checkers/base/basic_error_checker.py

```diff
@@ -204,6 +204,11 @@ class BasicErrorChecker(_BasicChecker):
             "which results in an error since Python 3.6.",
             {"minversion": (3, 6)},
         ),
+        "E0134": (
+            "Name %r is parameter and nonlocal",
```

Comment:
```suggestion
            "Name %r is both parameter and nonlocal",
```

Reply:
I'm good to change to `.. is both parameter and nonlocal`, but python says `.. is parameter and nonlocal`:
```python
>>> def tomato(is_tasty):
...     nonlocal is_tasty
...
  File "<stdin>", line 2
SyntaxError: name 'is_tasty' is parameter and nonlocal
```
wdyt?

Reply:
Hm, I'd say let's overwrite Python here and use more "readable" English. As pylint output is often read form IDEs I think we an be a little bit more verbose than the traditional error messages Python uses.

Reply:
Totally agree.